### PR TITLE
Proposal:  CloudEvents Spec in Chinese

### PR DESCRIPTION
**Pull request by JieDing**
## Proposed Changes

I'd like to translate a Chinese Version Specification based on the English one  in order to lower the difficulty of reading and accelerate the spread of CloudEvents for Chinese developers.
-Currently, these documents below have been done based on the spec v1.0.1 and v1.0:
-README.md, primer.md and spec.md.
-other documents in both versions can be started later if necessary.
@duglin

### CODE REVIEW

**Comment by JieDing** [comment]
@duglin

**Comment by duglin** [comment]
It's on the agenda for the call today.
Also, can you re-base, I think you have old commits in your list

**Comment by JieDing** [comment]
> It's on the agenda for the call today. Also, can you re-base, I think you have old commits in your list

Thanks a lot for replying. I'll re-base it asap.

**Comment by duglin** [comment]
@JieDing the problem isn't the files, the problem is you have old (other people's) commits as part of this PR. You should rebase on top of the latest version of the branch (`main`) and only include your commits

File: Chinese-Manual/README.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+# CloudEvents 中文规范
+
+![CloudEvents logo](https://github.com/cncf/artwork/blob/master/projects/cloudevents/horizontal/color/cloudevents-horizontal-color.png)
+
+Declaration： This manual aims to provide a fast and brief introduction of CloudEvents
+in Chinese for people who are new to CloudEvents.
+Most of the content is translated from the original English version.
+It is strongly recommended to read the English version if you find anything lost in translation.
+
```
Comment (cathyhongzhang) [inline Chinese-Manual/README.md:9]: This section describes why we need CloudEvents in Chinese. Suggest adding a section title (both English and Chinese) to this section.

File: Chinese-Manual/spec_CN.md [hunk 2]
```diff
@@ -0,0 +1,482 @@
...
+总体来看，这种匮乏严重阻碍了事件数据的可移植性和生产力。
+
+CloudEvents是一个以通用格式来描述事件数据的标准。它提供了事件在服务、平台和系统中的互操作性。
+
+事件格式指定如何使用某些编码格式序列化一个 CloudEvent。
+支持那些编码且兼容 CloudEvents 的实现必须遵守相应事件格式中指定的编码规则。
+所有实现都必须支持 [JSON 格式](json-format.md)。
+
+有关规范背后的历史、开发和设计原理等更多信息，
+请参阅 CloudEvents  [入门文档](primer_CN.md)。
```
Comment (cathyhongzhang) [inline Chinese-Manual/spec_CN.md:38]: These links are broken when I clicked them. Not sure if the issue is specific to me.

File: Chinese-Manual/README.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+被选为云原生沙箱级项目。
+
+## CloudEvents 文件
+
+现有文件如下:
+
+|                               |                                 最新版本                                  |                                      工作草稿                                       |
+| :---------------------------- | :-----------------------------------------------------------------------------: | :--------------------------------------------------------------------------------------: |
+| **核心标准:**       |
+| CloudEvents                   |          [v1.0.1](https://github.com/cloudevents/spec/blob/v1.0.1/spec.md)          |            [master](https://github.com/cloudevents/spec/blob/master/spec.md)             |
```
Comment (duglin) [inline Chinese-Manual/README.md:35]: I think the links here are old since the branch is now "main"

**Comment by duglin** [comment]
@JieDing thanks a ton for doing this. We discussed this on last week's call and there were just a few comments:
- are you planning on keeping these documents up to date as we make changes?
- can you put them under `cloudevents/translated/zh-cn/`  ? Is "zh-cn" the right code?

**Comment by JieDing** [comment]
> * zh-cn


> @JieDing thanks a ton for doing this. We discussed this on last week's call and there were just a few comments:
> 
> * are you planning on keeping these documents up to date as we make changes?
> * can you put them under `cloudevents/translated/zh-cn/`  ? Is "zh-cn" the right code?

* Yes, I'll keep translating documents as you make changes. 
* BTY,  please @me to let me know if there are some documents you prefer to translate first.
* Documents would be translated based on my schedule if there is no specified order of translation.
* I'll put documents under `cloudevents/translated/zh-cn/` later. "zh-cn" is the right code.

**Comment by JieDing** [comment]
This PR seems doesn't meet the verify checker as it [said](https://github.com/cloudevents/spec/runs/4228594241?check_suite_focus=true) "Can't find section '#XXX' in ./cloudevents/translated/zh-cn/primer_CN.md" where XXX are Chinese characters.
I was wondering if the href checker doesn't support Chinese since I can find and jump to those sections in my repository. 
@duglin

**Review by ifplusor — CHANGES_REQUESTED** [review]

**Comment by duglin** [comment]
@JieDing did you want to address the comments made?

**Comment by JieDing** [comment]
> @JieDing did you want to address the comments made?
I've made some changes to see if it's gonna address the problem

File: cloudevents/translated/zh-cn/README_CN.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+| CloudEvents SDK Requirements  |                                        -                                        |             [WIP](https://github.com/cloudevents/spec/blob/main/cloudevents/SDK.md)             |
+| Documented Extensions         |                                        -                                        |    [WIP](https://github.com/cloudevents/spec/blob/main/cloudevents/documented-extensions.md)    |
+| Primer                        |         [v1.0.1](https://github.com/cloudevents/spec/blob/v1.0.1/primer.md)         |           [WIP](https://github.com/cloudevents/spec/blob/main/cloudevents/primer.md)            |
+| Proprietary Specifications    |                                        -                                        |      [WIP](https://github.com/cloudevents/spec/blob/main/cloudevents/proprietary-specs.md)      |
+
+对于初次接触CloudEvents的同学，可以通过阅读[入门文档](primer_CN.md)增加对CloudEvents规范的目标和设计理念
+的总体理解，
+希望在最短时间内使用CloudEvents 规范的同学，可以直接阅读[核心规范](spec_CN.md)。
+
+由于并非所有事件生产者都默认生产符合CloudEvents规范的事件，因此可以用[CloudEvents适配器](https://github.com/cloudevents/spec/blob/main/cloudevents/adapters.md)
```
[the remark this answers is not in the document]
Comment (duglin) [reply cloudevents/translated/zh-cn/README_CN.md:54]: I kind of like the original version better.

**Comment by duglin** [comment]
@JieDing thanks! I'll add it to this week's agenda for final approaval

**Comment by JieDing** [comment]
> @JieDing thanks! I'll add it to this week's agenda for final approaval

I do feel happy to make a contribution to CloudEvents. Just let me know if anything needs to be modified.

**Comment by duglin** [comment]
Approved on the 12/02 call
thanks again @JieDing !!

**Pull request merged**